build.sh

```diff
@@ -222,10 +227,15 @@ while [[ $# -gt 0 ]]; do
         ICU_PATH=$1
         # `eval` used to resolve tilde in the path
         eval ICU_PATH="${ICU_PATH:6}"
-        if [[ ! -d $ICU_PATH || ! -d $ICU_PATH/unicode ]]; then
-            # if --custom-icu is given, do not fallback to no-icu
-            echo "!!! couldn't find ICU at $ICU_PATH"
-            exit 1
+        if [[ ! -d "$ICU_PATH" || ! -d "$ICU_PATH/unicode" ]]; then
+            ICU_PATH_LOCAL="$CHAKRACORE_DIR/$ICU_PATH"
```

Comment:
please define a bash method and use it instead of using multiple copies of the same code.

Reply:
Sure